DrugResistance/src/main/java/edu/stanford/hivdb/drugresistance/database/ConditionalComments.java

```diff
@@ -167,22 +167,22 @@ protected BoundComment(
 		public Mutation getBoundMutation() { return mutation; }
 		public Gene getGene() { return gene; }
 		public DrugClass drugClass() { return drugClass; }
-
-//		@Deprecated
-//		public String getConsensus() {
-//			if (mutation != null) {
-//				return mutation.getConsensus();
-//			}
-//			return null;
-//		}
-
-//		@Deprecated
-//		public String getTriggeredAAs() {
-//			if (mutation != null) {
-//				return mutation.getAAs();
-//			}
-//			return null;
-//		}
+		
+		@Deprecated
```

Comment:
If my dataFetcher implementation in GraphQL look ok, I'll delete these methods.

Reply:
LGTM.